feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -184,19 +188,35 @@ fun PreviewScreen(
                         )
                     }
 
-                    TestingButton(
+                    Row(
                         modifier = Modifier
-                            .testTag("ToggleCaptureMode")
                             .align(Alignment.TopEnd)
                             .padding(12.dp),
-                        onClick = { viewModel.toggleCaptureMode() },
-                        text = stringResource(
-                            when (previewUiState.currentCameraSettings.captureMode) {
-                                CaptureMode.SINGLE_STREAM -> R.string.capture_mode_single_stream
-                                CaptureMode.MULTI_STREAM -> R.string.capture_mode_multi_stream
-                            }
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        TestingButton(
+                            modifier = Modifier
+                                .testTag("ToggleCaptureMode"),
+                            onClick = { viewModel.toggleCaptureMode() },
+                            text = stringResource(
+                                when (previewUiState.currentCameraSettings.captureMode) {
+                                    CaptureMode.SINGLE_STREAM -> R.string.capture_mode_single_stream
+                                    CaptureMode.MULTI_STREAM -> R.string.capture_mode_multi_stream
+                                }
+                            )
                         )
-                    )
+                        if (previewUiState.currentCameraSettings.supportedStabilizationMode !=
+                            SupportedStabilizationMode.UNSUPPORTED &&
+                            previewUiState.currentCameraSettings.videoCaptureStabilization ==
```

Comment:
I know we're using the same icon here for `ON` and `HIGH_QUALITY`, but we should at least change the `contentDescription` to specify whether `ON` or `HIGH_QUALITY` is selected. In that case, you'll also need to check `previewStabilization`.